docs/en/Running-Spark-on-Alluxio.md

```diff
@@ -88,7 +88,8 @@ When you have a running Spark cluster (or Spark standalone), you can run the fol
 $ checker/bin/alluxio-checker.sh spark <spark master uri> <spark partition number(optional)>
 ```
 
-This will report potential problems that might prevent you from running Spark on Alluxio. 
+You can use `-h` to display the helpful information of this command.
```

Comment:
grammar: either "to display helpful information about the command" or "to display the help information for the command"